core/src/main/scala/cats/Decidable.scala

```diff
@@ -0,0 +1,71 @@
+package cats
+
+import cats.data.INothing
+
+import scala.annotation.implicitNotFound
+
+/**
+ * [[Decidable]] functors are functors that supply
+ * a `decide` operation allowing choices to be made.
+ *
+ * This is comparable to [[Alternative]] in the
+ * covariant case.
+ *
+ * Must obey laws in cats.laws.DecidableLaws.
+ *
+ * Based on ekmett's contravariant library:
+ * https://hackage.haskell.org/package/contravariant-1.4/docs/Data-Functor-Contravariant-Divisible.html#g:2
+ */
+@implicitNotFound("Could not find an instance of Decidable for ${F}")
```

Comment:
We stopped using these, they were causing misleading compiler errors.

Reply:
Interesting! I guess I could see how that would happen with enough transitive chains of implicit resolution

Reply:
Context:
- https://github.com/typelevel/cats/issues/4059